libs/components/lookup/src/lib/modules/lookup/lookup.component.ts

```diff
@@ -204,6 +204,12 @@ export class SkyLookupComponent
   public addClick: EventEmitter<SkyLookupAddClickEventArgs> =
     new EventEmitter();
 
+  /**
+   * @internal
+   */
+  @Output()
+  public lookupOpenChange = new EventEmitter<boolean>();
```

Comment:
We need to call `.complete()` on this event emitter in the `ngOnDestroy` lifecycle hook.

Reply:
Good call. Added.